Revise `README.md`. (#169)
graalvm/setup-graalvm: README.md

# GitHub Action for GraalVM [![CI](https://github.com/graalvm/setup-graalvm/actions/workflows/ci.yml/badge.svg)](https://github.com/graalvm/setup-graalvm/actions/workflows/ci.yml)

> Set up your GitHub Actions workflow with a specific [GraalVM][graalvm] distribution, and use it both as your JDK and to build [efficient native images][graalvm].

Author: ```suggestion
Set up your GitHub Actions workflow with a specific [GraalVM][graalvm] distribution, and use it both as your JDK and for [ahead-of-time Native Image compilation][graalvm].
```
